src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -598,8 +598,10 @@ static String getResource(String key) {
                     "An error occurred when matching VALUES list to table columns. Please verify SQL syntax."},
             {"R_invalidValuesList", "An error occurred when reading VALUES list. Please verify SQL syntax."},
             {"R_enclaveNotSupported", "The SQL Server instance does not support enclave based computations."},
-            {"R_enclaveNoAttestationProtocol",
-                    "The \"enclaveAttestationProtocol\" connection property must be specified with \"enclaveAttestationUrl\"."},
+            {"R_enclaveAEdisabled",
```

Comment:
No need for 2 different error strings here. You can combine them into one string, something like `\"columnEncryptionSetting\",  \"enclaveAttestationUrl\", \"enclaveAttestationProtocol\" properties are required to enable Always Encrypted with secure enclaves.`